Minor fixes to readme

thekaranacharya opened the pull request:
Noticed some errors, fixed them for 0.4.2

thekaranacharya committed:
Minor fixes

ShreyaR commented on README.md:
````diff
@@ -80,8 +81,8 @@ pip install guardrails-ai
     ```bash
-    guardrails hub install hub://guardrails/competitor_check
-    guardrails hub install hub://guardrails/toxic_language
+    guardrails hub install hub://guardrails/regex_match
````
@thekaranacharya what's the reason for the switch to `regex_match` and `valid_length`?

thekaranacharya replied:
@ShreyaR Below we were importing and using `RegexMatch` and `ValidLength`, hence. The installed `competitor_check` and `toxic_language` were not actually being used.

CalebCourier replied:
@ShreyaR looks like in the current README we show installing `competitor_check` and `toxic_language` but then below that show using `regex_match` and `valid_length`.  This change synchronizes the install with the usage

ShreyaR replied:
Ah great catch! In that case can we update the example usage so that we use the CompetitorCheck and ToxicLanguage validators?

The reason for biasing towards these validators is that they're good starter examples for showing useful but technically complex validators. We've received feedback in the past that the structural validators undermine how complex validators can generally be.

thekaranacharya replied:
Sure, that sounds right. Updating.

thekaranacharya committed:
Update readme with correct fixes and working examples

thekaranacharya commented:
Hello @ShreyaR and @CalebCourier, added examples for using `ToxicLanguage` and `CompetitorCheck` and also noticed few bugs in the `RegexMatch` example, corrected them. Also added console outputs so that user can see what is the expected output when a guardrail fails with what error message.
